tools/activate_emsdk.py

```diff
@@ -16,7 +16,7 @@
 EMSDK_PATH = os.path.join(EMSDK_ROOT, 'emsdk.py')
 
 # See lib/web_ui/README.md for instructions on updating the EMSDK version.
-EMSDK_VERSION = '3.1.3'
+EMSDK_VERSION = '3.1.36'
```

Comment:
Skia seems to be on 3.1.15 and https://github.com/flutter/engine/blob/main/lib/web_ui/README.md#upgrading-the-emscripten-sdk-for-the-canvaskit-build advises to keep in sync with Skia

Reply:
After some investigation, 3.1.15 has a bug that breaks skwasm compilation. 3.1.32 seems to work for both CanvasKit and skwasm, so let's try using that one.

Reply:
Skia is still on 3.1.15, right? Is it not a concern?

Reply:
It's not an actual concern. Keeping the version of the emsdk exactly in sync is not necessary.